Touch Bar Simulator/Info.plist

```diff
@@ -30,6 +30,22 @@
 	<string>MIT License © Sindre Sorhus</string>
 	<key>NSPrincipalClass</key>
 	<string>NSApplication</string>
+	<key>NSServices</key>
+	<array>
+		<dict>
+			<key>NSKeyEquivalent</key>
+			<dict/>
```

Comment:
Yes, I currently use ⌃⌥⌘T, but I added it in SysPrefs. I don't know how to add such a keystroke into plist file. [Apple reference](https://developer.apple.com/library/content/documentation/Cocoa/Conceptual/SysServices/Articles/properties.html#//apple_ref/doc/uid/20000852-CHDJDFIC) only covers ⌘ keystrokes (or at most ⇧⌘ ones), as it says:

> NSKeyEquivalent is an optional dictionary that specifies the keyboard equivalent for invoking the menu command. Like NSMenuItem, the only entry in the dictionary should have the key default with a string value that can be localized in the ServicesMenu.strings file. The string value must be a single character. The keyboard shortcut is this one character, with the Command key modifier. If the character is uppercase, the Shift modifier is also used.
Use key equivalents sparingly. Remember that your shortcuts are being added to the collection of shortcuts defined by each application as well as defined by the other services. When an application already has a shortcut with that key equivalent, the application’s shortcut wins. If multiple services define the same shortcut, which one gets invoked is undefined.

Since Finder already uses ⌘T, I decided to leave it blank, so everyone choses his own keystroke.